concent_api/core/views.py

```diff
@@ -36,7 +36,7 @@ def send(request, client_message, client_public_key):
 @require_golem_auth_message
 @handle_errors_and_responses
 @require_POST
-def receive(_request, message):
+def receive(_request, message, _client_public_key):
```

Comment:
I see that in `receive` and `receive_out_of_band` endpoints,`client_public_key` from message is used to update subtasks, whereas in `send`, the one that comes from wrapper. What is the reason for that?